docs/source/release-notes.rst

```diff
@@ -65,10 +65,10 @@ Utils
 
 Docs
 ^^^^
-- Added an :doc:`Environments </environments>` page which outlines how
+- Added an :ref:`Environments page <environments>` that outlines how
```

Comment:
These `:doc:` links were broken (discovered this from last release while editing here). Another reason to prefer the `:ref:` links that I added here